tests/functional/lang/eval-fail-bad-string-interpolation-2.err.exp

```diff
@@ -1 +1 @@
-error: getting status of '/pwd/lang/fnord': No such file or directory
+error: path '/pwd/lang/fnord' does not exist
```

Comment:
This isn't a sign we lost a potentially useful `perror()` via `SysError` is it?

Reply:
No, it's because `SourceAccessor::lstat()` now handles the file-not-found case. `SourceAccessor::maybeLstat()` returns `std::nullopt` if the file doesn't exist, but throws a `SysError` for everything else.